python/tvm/relay/tensorrt.py

```diff
@@ -718,7 +749,41 @@ def visit_call(self, call):
         return super().visit_call(call)
 
 
-def PruneSubgraphs(mod, compiler="tensorrt", use_implicit_batch=True, prune_no_macs=False):
+class ComputeHeavyGraph(ExprMutator):
+    """
+    Visits the Graph recursively and checks if it contains compute heavy ops like convolutions and
+    its transpose, dense and batch mat-mul.
+    """
+
+    def __init__(self):
+        ExprMutator.__init__(self)
+        self.is_compute_heavy = False
+
+    def visit_call(self, call):
+        heavy_ops = set(
```

Comment:
compute_intensive might be better name than compute_heavy

Reply:
Done